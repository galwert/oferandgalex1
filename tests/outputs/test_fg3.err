smash error: waitpid failed: No child processes
smash error: waitpid failed: No child processes
smash error: waitpid failed: No child processes
smash error: waitpid failed: No child processes
smash error: waitpid failed: No child processes
smash error: kill failed: No such process
