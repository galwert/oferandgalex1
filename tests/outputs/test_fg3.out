smash> smash> smash> smash> smash> smash> sleep 300& : 3
smash: got Ctrl-C
smash: process 6594 was killed
smash> [1] sleep 100& : 6592 0 secs
[2] sleep 200& : 6593 0 secs
[3] sleep 300& : 6594 0 secs
[4] sleep 400& : 6595 0 secs
[5] sleep 500& : 6596 0 secs
smash> sleep 100& : 1
smash: got Ctrl-C
smash: process 6592 was killed
smash> [1] sleep 100& : 6592 1 secs
[2] sleep 200& : 6593 1 secs
[3] sleep 300& : 6594 1 secs
[4] sleep 400& : 6595 1 secs
[5] sleep 500& : 6596 1 secs
smash> sleep 200& : 2
smash: got Ctrl-C
smash: process 6593 was killed
smash> [1] sleep 100& : 6592 1 secs
[2] sleep 200& : 6593 1 secs
[3] sleep 300& : 6594 1 secs
[4] sleep 400& : 6595 1 secs
[5] sleep 500& : 6596 1 secs
smash> sleep 500& : 5
smash: got Ctrl-C
smash: process 6596 was killed
smash> [1] sleep 100& : 6592 2 secs
[2] sleep 200& : 6593 2 secs
[3] sleep 300& : 6594 2 secs
[4] sleep 400& : 6595 2 secs
[5] sleep 500& : 6596 2 secs
smash> sleep 500& : 5
smash> smash: got Ctrl-Z
